src/containers/About/About.js

```diff
@@ -118,7 +119,7 @@ export /* istanbul ignore next */ class About extends Component {
     if (dashboardInfo && !loading) {
       propertiesToCheck.forEach(property => {
         const value = this.getDisplayValue(dashboardInfo[property]);
-        if (value) {
+        if (value && value !== 'Not Installed') {
```

Comment:
can we send an empty string / null back instead?

Reply:
An empty string implies that triggers is installed 
Which brings up an error saying that it cant find triggers version which I wanted to avoid if a user doesn't have triggers installed

Reply:
I'm not sure I follow, why would an empty string imply that it's installed? Are you referring to the current implementation or a user's expectations?

The API should not have to return a magic string to satisfy the current implementation in the front end, instead we should design the API in a way that makes sense (either returning null / empty string, or omitting the field entirely) and update the front end code accordingly.